src/eval/index.php

```diff
@@ -1,9 +1,12 @@
 <?php
+	// set me to false to turn off sandboxing
+	$sandbox_me = true;
```

Comment:
This forces users who want to locally serve the repl to modify the code. It's not a huge modification, but the environment variable solution avoids this. Was there a particular reason why you went the boolean route? Just curious.